decorator.rb

```diff
@@ -0,0 +1,37 @@
+require './nameable'
+require './person'
+
+class Decorator < Nameable
```

Comment:
- [ ] Only the correct_name method worked on the person instance, however, we are required to make sure the `capitalize` and `trimmer` decorator works as well. As we can see in the screenshot below
![Screenshot from 2023-06-28 20-10-15](https://github.com/osmanwako/schoolibrary/assets/46388113/c387bd0b-77e1-4b48-863e-9da201e1272e)
Let us copy the content of the screenshot below in our decorator class, then run `ruby decorator.rb` to make sure the test pass before submitting another code review request
![Screenshot from 2023-06-28 20-13-28](https://github.com/osmanwako/schoolibrary/assets/46388113/320e2d94-1597-41be-9633-1ce10320fac3)